blenderproc/python/types/MeshObjectUtility.py

```diff
@@ -671,9 +687,11 @@ def create_primitive(shape: str, **kwargs) -> "MeshObject":
         bpy.ops.mesh.primitive_cone_add(**kwargs)
     elif shape == "PLANE":
         bpy.ops.mesh.primitive_plane_add(**kwargs)
-    elif shape == "SPHERE":
+    elif "SPHERE" in shape:
```

Comment:
Why is icosphere not just a another elif?

Reply:
I though of icosphere as a special case of sphere, so we would get uv sphere per default. Otherwise it should probably be `UV_SPHERE` and `ICO_SPHERE`.

Reply:
Yes but at the moment for ICO_SPHERE two sphere obejcts are actually created.
I think just having separate `UV_SPHERE` and `ICO_SPHERE` would be the better solution, with allowing `SPHERE` next to `UV_SPHERE` for backwards compatibility.

Reply:
Oops, yes, you are right, we don't want two spheres in case of icosphere. Ok, so
```python
elif shape in ["SPHERE", "UV_SPHERE"]
elif shape == "ICO_SPHERE"
```
it is.

Reply:
Yes perfect :+1: